extensions/classifiers/LDAStringClassifier/LDAStringClassifier.py

```diff
@@ -627,9 +690,7 @@ def validate(self, classifier_data):
             '_w_dp',
             '_l_dp',
             '_c_dl',
-            '_c_lw'
-        ]
-        list_of_list_bool_properties = [
+            '_c_lw',
             '_b_dl'
```

Comment:
Ah, I see. 0/1 booleans are still conceptually booleans, though (albeit not Python booleans). So if we expect the array to contain only 0s and 1s, then there should be additional validation to check that the only allowed values are 0 and 1.